src/main/java/com/example/bank/service/demo/entity/Account.java

```diff
@@ -0,0 +1,45 @@
+package com.example.bank.service.demo.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+
+@Entity
+@AllArgsConstructor
```

Comment:
the entity should also have no-args constructor - add this annotation as well

Reply:
done